setup.sh

```diff
@@ -8,29 +8,25 @@ if [[ $HOSTNAME == "login.snowmass21.io" ]]; then
     module purge
     module use /software/modulefiles
  
-    #module load python/2.7.15
-    #module load py-numpy/1.15.2-py2.7
-    #module load py-six/1.11.0-py2.7
-    
     module load python/3.7.0
     module load gcc-8.2.0
     module load cmake
     
     module load py-numpy/1.15.2-py3.7
     module load py-six/1.11.0-py3.7
     
-    source /cvmfs/sft.cern.ch/lcg/views/LCG_101/x86_64-centos7-gcc8-opt/setup.sh
-    
-    #convenient
     module load emacs
-else
-    which root >> /dev/null
-    if [[ $? -ne 0 ]]; then
-	echo "Please ensure ROOT is available and retry"
-	exit
-    fi
+# else
+#     which root >> /dev/null
+#     if [[ $? -ne 0 ]]; then
+# 	echo "Please ensure ROOT is available and retry"
+# 	exit
+#     fi
 fi
 
+source /cvmfs/sft.cern.ch/lcg/releases/LCG_99/ROOT/v6.22.06/x86_64-centos7-gcc8-opt/ROOT-env.sh
```

Comment:
do we need the ROOT import here? it should come with the LCG_101 view

Reply:
im not sure.  I'll remove it and see if it causes trouble

Reply:
and note, I was using LCG_99 in my previous testing, so I stuck with that.  Is there any reason to use 101 instead?

Reply:
Ignore my last comment, I switched to 101